bundle/Resources/views/ContentType/field_types.html.twig

```diff
@@ -116,6 +116,25 @@
     </div>
 {% endblock %}
 
+{% block ezobjectrelationlist_field_definition_edit %}
+    <div class="ezobjectrelationlist-settings selection-default-location">
+        {{- form_label(form.selectionDefaultLocation) -}}
+        {{- form_errors(form.selectionDefaultLocation) -}}
+        {{- form_widget(form.selectionDefaultLocation) -}}
+
+        <button data-universaldiscovery-title="{{"field_definition.ezobjectrelationlist.selection_root_udw_title"|trans({}, "ezrepoforms_content_type")}}"
```

Comment:
The same as for ezrelation (https://github.com/ezsystems/repository-forms/pull/31#discussion_r33573600):
* please add container for displaying selected content's name
* please add `data-relation-selected-root-name-selector` attribute in button which will contain selector pointing on container with selected content's name